### DISCUSSION TITLE
After cloning the repo, please make a New Branch before editing the project

### DISCUSSION THREAD

**Discussion by BobbyCasas** [BobbyCasas/cs3443-app-prog-group12-pathfinders-maze#23] [Announcements]
Once you successfully clone the repo, be sure to make a new branch before editing anything. 
Right-click project folder Team > Switch To > New Branch
(PLEASE DO NOT EDIT AND PUSH THE MASTER BRANCH)

**Comment by gug903** [comment]
sorry my b

**Reply by gug903** [reply]
is there an issue or is it fine for now

**Reply by gug903** [reply]
you can still access the old version
